src/applications/vaos/routes.jsx

```diff
@@ -43,7 +43,7 @@ export default function createRoutesWithStore(store) {
             )}
           />
           <EnrolledRoute
-            path="/new-covid-19-vaccine-booking"
```

Comment:
I think we still need to determine if we for sure need to redirect the old url, but wouldn't hurt to add it in regardless: https://reactrouter.com/web/api/Redirect

Reply:
@jbalboni, the files are ready to merge with master now. Did you want me to do something differently to the file?

Reply:
@jenniemc We either need to confirm with Lauren that a redirect from the old urls is not needed, or go ahead and add a redirect in this PR.

Reply:
@laurenernest, is the redirect from the Old URL address, `/new-covid-19-vaccine-booking` to the New URL address, `/new-covid-19-vaccine-appointment` required? 

cc @jbalboni

Reply:
keep a redirect, but send to /health-care/schedule-view-va-appointments/appointments/new-appointment (similar to the non-covid pages)